tests/shared/src/test/scala/scala/meta/tests/parsers/LitSuite.scala

```diff
@@ -155,7 +155,7 @@ class LitSuite extends ParseSuite {
         Nil,
         Type.Name("Foo"),
         Nil,
-        Ctor.Primary(Nil, Name(""), Nil),
+        Ctor.Primary(Nil, Name(""), emptyParamClause),
```

Comment:
is this the same as `ctor`? same in all, or most, places below.

Reply:
Ach actually, I replace them with `EmptyCtor()` <- I think it is more explicit.